bin.src/mtoods

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+# This file is part of ctrl_oods
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import logging
+
+import lsst.log as lsstlog
+from lsst.ctrl.oods.mtoods_csc import MtOodsCsc
+
+lsstlog.usePythonLogging()
+
+LOGGER = logging.getLogger(__name__)
+F = "%(levelname) -10s %(asctime)s.%(msecs)03dZ %(name) -30s %(funcName) -35s %(lineno) -5d: %(message)s"
+logging.basicConfig(level=logging.INFO, format=(F), datefmt="%Y-%m-%d %H:%M:%S")
```

Comment:
Not quite sure what you need parentheses for when defining `format`. Doesn't `format=F` work just fine?

Reply:
I'm leaving this for now, because it's consistent with the other two commands (atoods and ccoods), but will add this to the things I'm cleaning up in another ticket.

Reply:
This will all get fixed when you start to use black.